nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/GenerateBashAutoCompletionCommand.java

```diff
@@ -106,8 +91,21 @@
             + "}\n";
     private static final String COMPLETE_CALL = "complete -F _asadmin asadmin";
     private static final String ADD_PATH = "PATH=$PATH:";
-    
     private static final String DEFAULT_FILE = File.separator + "bin" + File.separator + "bash_autocomplete";
+    private final static LocalStringsImpl strings = new LocalStringsImpl(GenerateBashAutoCompletionCommand.class);
+    private final Logger LOGGER = Logger.getLogger(GenerateBashAutoCompletionCommand.class.getName());
+    @Param(optional = true, primary = true, name = "file")
+    String filePath;
+    @Param(optional = true, defaultValue = "false")
+    Boolean force;
+    @Param(optional = true, defaultValue = "false")
+    Boolean localCommands;
+    @Inject
+    ServiceLocator habitat;
+    @Inject
+    ServerContext serverContext;
+    private CLIContainer cliContainer;
+    private File file;
```

Comment:
Can we have some white space around annotated variables please? 🤢

```suggestion
    
    @Param(optional = true, primary = true, name = "file")
    String filePath;
    
    @Param(optional = true, defaultValue = "false")
    Boolean force;
    
    @Param(optional = true, defaultValue = "false")
    Boolean localCommands;
    
    @Inject
    ServiceLocator habitat;
    
    @Inject
    ServerContext serverContext;
    
    private CLIContainer cliContainer;
    private File file;
```